app/components/blacklight/search_context/server_item_pagination_component.rb

```diff
@@ -12,35 +12,52 @@ def initialize(search_context:, search_session:, current_document:)
       end
 
       def render?
-        @search_context.present? && (@search_context[:prev] || @search_context[:next]) && (@search_session['document_id'] == @current_document_id)
+        return false unless @search_session['document_id'] == @current_document_id
+
+        return true if total == 1
+
+        @search_context.present? && (@search_context[:prev] || @search_context[:next])
+      end
+
+      def display_separator
+        '|' unless total == 1
```

Comment:
I would make this display separator a translation.

Reply:
I have tried to do that.